DataFormats/TrackingRecHitSoA/test/alpaka/Hits_test.cc

```diff
@@ -0,0 +1,47 @@
+#include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsHost.h"
+#include "DataFormats/TrackingRecHitSoA/interface/TrackingRecHitsDevice.h"
+#include "DataFormats/TrackingRecHitSoA/interface/alpaka/TrackingRecHitsSoACollection.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/devices.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/host.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/memory.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaInterface/interface/workdivision.h"
+
+#include "Geometry/CommonTopologies/interface/SimplePixelTopology.h"
+
+#include <alpaka/alpaka.hpp>
+#include <unistd.h>
+
+using namespace ALPAKA_ACCELERATOR_NAMESPACE;
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+  namespace testTrackingRecHitSoA {
+
+    template <typename TrackerTraits>
+    void runKernels(TrackingRecHitSoAView<TrackerTraits>& hits, Queue& queue);
+
+  }
+}  // namespace ALPAKA_ACCELERATOR_NAMESPACE
+
+int main() {
+  const auto host = cms::alpakatools::host();
+  const auto device = cms::alpakatools::devices<Platform>()[0];
+  Queue queue(device);
+
+  // inner scope to deallocate memory before destroying the queue
+  {
+    uint32_t nHits = 2000;
```

Comment:
are these const?

Reply:
Yes, could be even `constexpr` actually.

Reply:
Well, the compiler knows if a variable is going to change or not anyway.

Unless they need to be used in a constant evaluation expression (_e.g._ a template argument or array size), I think it doesn't really make any difference...